<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>751</width>
     <height>701</height>
    </rect>
   </property>
   <property name="title">
    <string>Img Show</string>
   </property>
   <widget class="QLabel" name="Label_Img_Show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>701</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>541</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Annotation </string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pose Model</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Cbox_pose_model">
       <item>
        <property name="text">
         <string>HRnet_w48_256x192</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Cbox_det_model">
       <item>
        <property name="text">
         <string>Yolox_s</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Detection Model:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_checkpoint_det">
       <property name="text">
        <string>Chekpoint</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_checkpoint_pose">
       <property name="text">
        <string>Checkpoint</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detect_vid">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_last">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove last value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLineEdit" name="LEdit_det_thr"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>keypoint score thr</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LEdit_kpt_thr"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>det score thr:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Export File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Edit_file_name"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="Label_Edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Label Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tolist">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>To list</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="slider_frame_no">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>750</y>
     <width>671</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEdit_frame_no">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Text_display">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>620</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
